README.md

```diff
@@ -452,7 +452,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
 *Libraries for using OpenGL in Go.*
 
 * [gl](https://github.com/go-gl/gl) - Go bindings for OpenGL. Requires an external dependency GLEW.
-* [glfw3](https://github.com/go-gl/glfw3) - Go bindings for GLFW 3.
+* [glfw3](https://github.com/go-gl/glfw) - Go bindings for GLFW 3.
```

Comment:
Thanks for catching this. It should also be renamed (and re-ordered) to `glfw` now.